4.26/platform_isv.html

```diff
@@ -38,6 +38,81 @@ <h2>Platform and Equinox API</h2>
   <tr>
     <td id="Platform" class="section" colspan="2"><h2>Platform Changes</h2></td>
   </tr>
+  <tr id="JobManager_Implementation"> 
+  <!-- https://bugs.eclipse.org/bugs/show_bug.cgi?id=574883 -->
+  <!-- https://github.com/eclipse-platform/eclipse.platform/commit/dc7bfe70caa56ad3a63b3181dad3364ec7498cf7 -->
+    <td class="title">JobManager implementation changes</td>
+    <td class="content">
+  <h2 id="possible-deadlock-in-ijobchangelistener">Possible deadlock in IJobChangeListener</h2>
+<p>Attention: The JobManager implementation was changed! The old
+JobMangager implementation notified IJobChangeListener about various
+IJobChangeEvent without strict order in various Threads. For example
+scheduled() was called in a Thread that did call schedule() on a Job
+while done() was normally called within a Worker Thread. In case of a
+repeated schedule() both Notifications could run in parallel. That was a
+race condition. The Listener could not deterministically know if the Job
+was still scheduled or already done. As a consequence a join() could
+have returned too early or the Progress View did not show Jobs that did
+still run. And probably many other things went wrong totally unnoticed.
+Even multiple notifications could have happened in parallel in various
+Worker Threads, as there is no guarantee that the next execution is done
+in the same Worker.</p>
+<p>To fix this indeterministic behavior all notifications for a Job will
+now be done one after the other. It is still not defined in which Thread
+the notification happens, but the new implementation will not call any
+IJobChangeListener until all pending events for the jobs are processed
+by all registered IJobChangeListener. The new implementation will also
+wait with any job state change until all calls to IJobChangeListener for
+that job returned.</p>
+<p>The IJobChangeListener javadoc clearly specifies &quot;whether the state
+change occurs before, during, or after listeners are notified is
+unspecified.&quot; - and the implementation changed within that broad
+specification. Unfortunately some IJobChangeListener around rely on the
+old implementation. They may now deadlock. For example the following
+snippet can now deadlock:</p>
+<pre><code> job.schedule(); <span class="hljs-comment">// may result in done();</span>
```

Comment:
I used an automated MarkDown to html converter. i am not into html. as long as it does not hurt i would leave it as is.

Reply:
So trust me in this case and remove all `<span ...>` and `</span>`, they make review unnecessary complicated and they use styles that aren't supported in any place where this file will be used because they need extra javascript loaded.